### ISSUE TITLE
support vue and or svelte.

### ISSUE THREAD

**Issue by susnick** [scopatz/nanorc#420]
Im looking for language support hat has a mix of other languages. Svelte would have javascript/typescript, html, css in one file. They have different comment characters for each type. Is this possible?

**Comment by davidhcefx** [comment]
Maybe you'll like to also ask at [galenguyer/nano-syntax-highlighting](https://github.com/galenguyer/nano-syntax-highlighting) and other forks!

**Comment by davidhcefx** [comment]
I don't think it is possible, because Nano currently only accepts one highlighting for the whole file. By the way, do you know any other text editor or plugins that has this ability?

**Comment by iconoclasthero** [comment]
I got to this repo because nano's not highlighting .vue files.  Just looking around to see if the files in this repo will syntax highlight .vue files without e.g., symlinking them to a .js extension and I found this thread at the top of the list...

Does this repo provide an easy way to get (what i assume i want is) js formatting for `*.vue` files?
Can I modify something to just add js support for `*.vue`?

**Comment by iconoclasthero** [comment]
Oh, I see, you can't just slap an extension on .vue files.  *^&%.

**Comment by iconoclasthero** [comment]
It seems creating /usr/share/nano/vue.nanorc with
```
#include "/usr/share/nano/xml.nanorc"
#Error in /usr/share/nano/vue.nanorc on line 1: Command "include" not allowed in included file
syntax "vue" "\.vue$"
# XML/HTML tags
color cyan "<[^>]*>"
# Vue directives
color magenta "\<(v-if|v-for|v-model|v-on|v-bind|v-text|v-html|@click|:key)\>"
# Vue specific attributes
color yellow "\<(ref|key|slot)\>"
# Strings
color yellow ""(\\.|[^"])*"" "'(\\.|[^'])*'"
# Comments
color blue "<!--" "-->"
# Vue interpolation
color brightwhite "\{\{[^}]*\}\}"
```
Is a marked improvement over what I had before, i.e., nothing.

**Comment by mofii** [comment]
I case it has any interest, I added a vue.nanorc file today in a different repo with this PR: [mofii/nano-syntax-highlighting/pull/18](https://github.com/mofii/nano-syntax-highlighting/pull/18)
Which was released with [v2.1.0](https://github.com/mofii/nano-syntax-highlighting/releases/tag/v2.1.0).
